Sources/TuistLoader/Models+ManifestMappers/SchemeDiagnosticsOptions+ManifestMapper.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import ProjectDescription
+import TSCBasic
+import TuistCore
+import TuistGraph
+
+extension TuistGraph.SchemeDiagnosticsOptions {
+    static func from(manifest: ProjectDescription.SchemeDiagnosticsOptions) -> TuistGraph.SchemeDiagnosticsOptions {
+        Self(rawValue: manifest.rawValue)
```

Comment:
minor: this works today as the options are 1-1 but may not be a safe mapping in general.

Reply:
restored old implementation